scheduler-portal/src/main/webapp/scheduler.conf

```diff
@@ -1,5 +1,8 @@
 sched.rest.url=http://localhost:8080/rest
```

Comment:
ideally, this could as well be inferred, by extracting the port and protocol from current url